src/lib.rs

```diff
@@ -13,47 +13,41 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
-//! # Kinematics (forward/inverse) library using [nalgebra](http://nalgebra.org).
-//!
-//! `k` has below functionalities
-//!
-//! 1. Forward kinematics
-//! 1. Inverse kinematics
-//! 1. URDF Loader
-//!
-//! See `Chain` as the top level interface.
-//!
 
+#![doc = include_str!("../README.md")]
 #![warn(missing_debug_implementations, rust_2018_idioms)]
 
-#[cfg(doctest)]
-doc_comment::doctest!("../README.md");
-
 mod chain;
 mod errors;
 mod funcs;
 mod ik;
-use nalgebra as na;
+
 pub mod iterator;
 pub mod joint;
 pub mod link;
 pub mod node;
 pub mod prelude;
 pub mod urdf;
 
-pub use self::chain::*;
-pub use self::errors::*;
-pub use self::funcs::*;
-pub use self::ik::*;
-pub use self::joint::{Joint, JointType};
-pub use self::link::Link;
-pub use self::node::{Node, NodeBuilder};
+pub use crate::{
+    chain::*,
+    errors::*,
+    funcs::*,
+    ik::*,
+    joint::{Joint, JointType},
+    link::Link,
+    node::{Node, NodeBuilder},
+};
 
 // re-export from nalgebra
 // include Real for backwards compatibility purposes
 // (na::Real used to be the name, so we used to re-export k::Real)
-pub use na::{Isometry3, RealField as Real, RealField, Translation3, UnitQuaternion, Vector3};
+#[doc(no_inline)]
+pub use nalgebra::{
+    Isometry3, RealField as Real, RealField, Translation3, UnitQuaternion, Vector3,
+};
 // export everything
 pub use nalgebra;
 pub use simba;
+#[doc(no_inline)]
 pub use simba::scalar::{SubsetOf, SupersetOf};
```

Comment:
This change makes it clearer where the item came from.

<img width="457" alt="doc" src="https://user-images.githubusercontent.com/43724913/147534585-4f05b7a3-145a-4c83-abab-9834bae80e1c.png">